src/temp-imported-plugins-event-names-temp.ts

```diff
@@ -0,0 +1,34 @@
+export const DownloadEvent = {
```

Comment:
what is this file for ?

Reply:
Currently temporarily as a result of a refactor there is an incompatibility in the form of the import so currently plugins events has been moved to hard coded temporarily